pages/myforms/[[...path]].tsx

```diff
@@ -94,6 +87,12 @@ export default function RenderMyForms({ templates }: MyFormsProps) {
       <TabPanel id="tabpanel-all" labeledbyId="tab-all" isActive={path === "all"}>
         <CardGrid cards={templatesAll}></CardGrid>
       </TabPanel>
+
```

Comment:
Moved outside of Tab and Tabpanel to avoid confusion from keyboard users. Otherwise keyboard tabing from a Tab the user would hit the create link and assume that is the Tabpanel.
The CSS just keeps the create link visually in the same spot.